app/controllers/casa_cases_controller.rb

```diff
@@ -157,17 +159,14 @@ def casa_case_params
       :court_report_due_date,
       :empty_court_date,
       contact_type_ids: [],
-      court_dates_attributes: [:date]
+      court_dates_attributes: [:date],
+      case_assignments_attributes: [:volunteer_id]
     )
   end
 
   def casa_case_create_params
-    create_params = if court_date_unknown?
-      casa_case_params.except(:court_dates_attributes)
-    else
-      casa_case_params
-    end
-
+    create_params = casa_case_params
+    create_params = create_params.except(:court_dates_attributes) if court_date_unknown?
     create_params.except(:empty_court_date)
```

Comment:
```suggestion
    create_params = casa_case_params.except(:empty_court_date)
    create_params = create_params.except(:court_dates_attributes) if court_date_unknown?
```
:man_shrugging:

Reply:
Might have been a bad call on my part since now the tests are failing :slightly_smiling_face:

Reply:
Ah yeah this is just an ordering problem. I'll fix it now. No big deal.

Reply:
I've reverted the commit.